server/api/payment.js

```diff
@@ -133,6 +133,21 @@ router.post('/payment', async (req, res, next) => {
       LIKECOIN.LIKE_COIN_ADDRESS,
       txData,
     );
+    if (!toSubscriptionURL) {
+      res.json({ txHash });
+    }
```

Comment:
sending both `res.json({ txHash });` here and below in 168 will cause error, on the other hand, why need to send early？

Reply:
Updated. This make sure sending res after `logTransferDelegatedTx` if user has `toSubscriptionURL`. If send earlier than that, oice cannot query the tx through tx API

Reply:
just defer the `res.json({ txHash })` after `logTransferDelegatedTx` i guess

Reply:
Updated